elements/pfe-styles/src/pfe-base.scss

```diff
@@ -82,33 +82,30 @@ th {
 // }
 
 body {
-  font-family: pfe-var(font-family);
-  font-size:   var( --pf-c--FontSize, pfe-var(content--FontSize));
-  font-weight: pfe-var(font-weight--light);
-  line-height: pfe-var(line-height);
+  @extend %pfe-content-base;
+  font-size: var( --pf-c--FontSize, pfe-var(content--FontSize));
   color: pfe-var(text);
   text-align: left;
   margin: 0;
 }
 
 // HTML TYPOGRAPHY
 .pfe-c-content {
-
   // HEADINGS
-  @each $heading-size in (h1, h2, h3, h4, h5, h6) {
-    #{$heading-size}  {
-        @extend %#{$heading-size}; 
+  @each $element in $pf-content-set {
```

Comment:
This feels too abstract to me - to find out what any of these values are you have to go search them out in another package entirely (pfe-sass vs. pfe-styles).  I would prefer more transparency and readability.

Reply:
👍